pkg/controller/operators/olm/operator.go

```diff
@@ -212,6 +214,19 @@ func newOperatorWithConfig(ctx context.Context, config *operatorConfig) (*Operat
 			return nil, err
 		}
 
+		subInformer := extInformerFactory.Operators().V1alpha1().Subscriptions()
```

Comment:
Now that we've done this, we need to get moving on combining subscription/csv so we don't have to cache both objects in both operators